geoips/interface_modules/output_formats/unprojected_image.py

```diff
@@ -61,10 +61,6 @@ def unprojected_image(
     image_width = float(x_size) / dpi
     image_height = float(y_size) / dpi
 
-    # Replace fig.savefig frameon=False argument with facecolor="none"
```

Comment:
@jsolbrig @cpcamacho I guess we don't need to retain notes about matplotlib v3.1.0...........